src/components/dashboard/ActivityRewardsSection/index.tsx

```diff
@@ -0,0 +1,133 @@
+      fontWeight="bold"
+      display="flex"
+      gap={1}
+    >
+      {title}
+    </Typography>
+  )
+}
+
+const ActivityRewardsSection = () => {
+  const chainId = useChainId()
+  const [matchingApps] = useRemoteSafeApps(SafeAppsTag.SAFE_GOVERNANCE_APP)
+  const isDarkMode = useDarkMode()
+
+  const governanceApp = matchingApps?.[0]
+  const today = Date.now()
+
+  const stepsActive = [
+    today >= CHAIN_START_TIMESTAMPS[chainId],
+    today >= JUNE_10_TIMESTAMP,
+    today >= SEPTEMBER_10_TIMESTAMP,
+  ]
+
+  const activityAppUrl = useShareSafeAppUrl(governanceApp?.url || '')
+
+  if (!governanceApp) return null
+
+  return (
+    <Grid item xs={12}>
+      <Accordion className={css.accordion} defaultExpanded>
+        <AccordionSummary
+          expandIcon={
+            <IconButton size="small">
+              <ExpandMoreIcon color="border" />
+            </IconButton>
+          }
+        >
+          <Typography component="h2" variant="subtitle1" fontWeight={700}>
+            Safe Activity App
+          </Typography>
+        </AccordionSummary>
+
+        <AccordionDetails>
+          <Card className={css.widgetWrapper}>
+            <Grid container xs={12} sx={{ gap: { xs: 4, md: 0 } }}>
+              <Grid item xs={12} md={6} p={0}>
+                <SvgIcon
+                  component={SafeMilesLogo}
+                  inheritViewBox
+                  color="border"
+                  className={classNames(css.milesIcon, { [css.milesIconLight]: !isDarkMode })}
+                />
+                <Typography
+                  variant="h1"
+                  fontWeight={700}
+                  mt={4}
+                  className={classNames(css.header, { [css.gradientText]: isDarkMode })}
+                >
+                  Interact with Safe and get rewards
+                </Typography>
+                <Button variant="contained" sx={{ mt: 3 }} href={activityAppUrl}>
+                  Open Activity App
+                </Button>
+              </Grid>
+              <Grid item xs={12} md={6} p={0}>
+                <Typography variant="overline" color="primary.light">
+                  How it works
+                </Typography>
+                <div className={css.steps}>
+                  <Step title="Lock SAFE to boost your miles!" active={stepsActive[0]} />
+                  <Step
+                    title={
+                      <>
+                        Get activity miles
+                        <Typography variant="caption" component="span" className={css.comingSoon}>
+                          Coming soon
+                        </Typography>
+                      </>
+                    }
+                    active={stepsActive[1]}
+                  />
+                  <Step title="Receive rewards" active={stepsActive[2]} />
+                </div>
+                <Link color="primary.main" fontWeight="bold" href={activityAppUrl}>
+                  Read more about the program
```

Comment:
Not sure what the best place to link this is since the governance app doesn't contain an educational section about the program (yet) but also linking to the landing page might be weird to take the user away from the app? cc @schmanu

Reply:
I think this will link to the landing page.

Reply:
I made it a TODO for now until we know where to link to.